DXMainClient/DXGUI/Multiplayer/GameLobby/CnCNetGameLobby.cs

```diff
@@ -1893,6 +1896,42 @@ private void BroadcastGame()
             if (ProgramConstants.IsInGame && broadcastChannel.Users.Count > 500)
                 return;
 
+            string gameBroadcastingString = BuildGameBroadcastingString();
+
+            // @TODO: Sending ctcp for broadcasting may not be needed now, if its only informing cncnet lobby
+            broadcastChannel.SendCTCPMessage(gameBroadcastingString, QueuedMessageType.SYSTEM_MESSAGE, 20);
+
+            // @TODO: Perhaps we should only update the topic when something has changed?
+            connectionManager.SetChannelTopic(channel, gameBroadcastingString);
+        }
+
+        private string BuildGameBroadcastingString()
+        {
+            // @TODO: Do we need a way to allow games/mods to specify their additional game options they want to broadcast
+            // For now hardcoded bits
```

Comment:
Yeah this is a hack, I am quite negative about such hacks. We are trying to walk away from hardcoded per game hacks, this goes against that path.

The client already has options broadcasting for all players inside the lobby, it already builds the string for all the checkboxes, dropdowns etc., so you don't need to invent much, rather reuse. Could add a new INI tag for a game option that would indicate that it is broadcasted to the outside lobby and split options building from that function, and/or reorganize it's a bit since outside/inside options communication seems to be more and more similar with this kind of changes.